Code/Editor/EditorViewportSettings.cpp

```diff
@@ -288,6 +297,9 @@ namespace SandboxEditor
     void SetCameraSpeedScale(float speedScale)
     {
         AzToolsFramework::SetRegistry(CameraSpeedScaleSetting, speedScale);
+
+        AzToolsFramework::ViewportInteraction::ViewportSettingsNotificationBus::Broadcast(
+            &AzToolsFramework::ViewportInteraction::ViewportSettingNotifications::OnCameraSpeedScaleChanged, speedScale);
```

Comment:
question/suggestion: I don't think this should need to be added if we use an event like everything else does (see top of file), I know it's a bit more typing but at least things are then consistent for now, instead of mixing how notifications are sent?

Reply:
I have transitioned this to use AZ::Event.

Reply:
Perfect thanks!